LTSketchbook/Part Number/6000/6603/DC1304A_B/DC1304A_B.ino

```diff
@@ -0,0 +1,363 @@
+        }
+        break;
+      case 4:
+        spi_out = control_byte;
+        LTC6603_write(LTC6603_CS, &spi_out, (uint8_t)1);
+        break;
+      default:
+        Serial.println("Incorrect Option");
+        break;
+    }
+    Serial.print ("\Control byte 0x" + String(control_byte, HEX) + "\n");
+  }
+}
+
+void menu_2_set_gpo()
+{
+  int16_t user_command; // The user input command
+
+  while (1)
+  {
+    Serial.print(F("*************************\n\n"));
+    Serial.println(F("0-Set GPIO Low"));
+    Serial.println (F("1-Set GPIO High"));
+    Serial.print(F("m-Main Menu\n"));
+    Serial.print(F("\nEnter a Command: "));
+    user_command = read_int();                              // Read the single command
+    if (user_command == 'm')
+      return;
+    else
+      Serial.println(user_command);
+    Serial.println();
+    switch (user_command)
+    {
+      case 0:
+        control_byte = (control_byte & LTC6603_GPO_MASK) | LTC6603_GPO_LOW;
+        spi_out = control_byte;
+        LTC6603_write(LTC6603_CS, &spi_out, (uint8_t)1);
+        break;
+      case 1:
+        control_byte = (control_byte & LTC6603_GPO_MASK) | LTC6603_GPO_HIGH;
+        spi_out = control_byte;
+        LTC6603_write(LTC6603_CS, &spi_out, (uint8_t)1);
+        break;
+      default:
+        Serial.println (F("incorrect option"));
+        break;
+    }
+    Serial.print ("\Control byte 0x" + String(control_byte, HEX) + "\n");
+  }
+}
+
+void menu_3_shutdown()
+{
+  Serial.print(F("* The LTC6603 is powered down *\n"));
+  control_byte = (control_byte & LTC6603_ONOFF_MASK) | LTC6603_SHDN;
+  spi_out = control_byte;
+  LTC6603_write(LTC6603_CS, &spi_out, (uint8_t)1);
+  Serial.print ("\Control byte 0x" + String(control_byte, HEX) + "\n");
+
+}
+
+void menu_4_poweron()
+{
+  Serial.print(F("* The LTC6603 is turning on *\n"));
+  control_byte = (control_byte & LTC6603_ONOFF_MASK) | LTC6603_PRUP;
+  spi_out = control_byte;
+  LTC6603_write(LTC6603_CS, &spi_out, (uint8_t)1);
+  Serial.print ("\Control byte 0x" + String(control_byte, HEX) + "\n");
+
+}
+
+void menu_5_changeclockfrequency()
+{
+  
+}
+
+//! Prints the title block when program first starts.
+void print_title()
+{
+  (F("\n*****************************************************************\n"));
+  Serial.print(F("* DC1304A-A Demonstration Program                               *\n"));
+  Serial.print(F("*                                                               *\n"));
+  Serial.print(F("* This program demonstrates how to send data to the LTC6603.    *\n"));
+  Serial.print(F("*                                                               *\n"));
+  Serial.print(F("* Set the baud rate to 115200 and select the newline terminator.*\n"));
+  Serial.print(F("*                                                               *\n"));
+  Serial.print(F("*****************************************************************\n"));
+}
+
+//! Prints main menu.
+void print_prompt()
+{
```

Comment:
These menu options are different from the menu options printed from loop()

Reply:
It's ok.  this is the very first thing that shows up, only once.